ci/nixfmt

```diff
@@ -13,6 +13,8 @@ if [[ $* =~ (^|[[:space:]])+--fix([[:space:]]|$)+ ]]; then
     check=""
 fi
 
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
What happens here if we are not in a git repo? If it is simple to handle that case, maybe it is good to not just `cd` away somewhere.

Reply:
Yeah should be easy to handle. And should maybe use git from nix in case you are in some weird scenario where git isn't in PATH. Then there's "what if" for the case of having a Nedryland project with several git repos in. But I think it's impossible to do something fool proof until flakes are out of experimental (or are abandoned) so we have one way of reasoning about the project.

Reply:
Yeah, I think "guessing" `git` is a very good guess :)